### ISSUE TITLE
TiledLayer<any> in TiledMapAbstract?

### ISSUE THREAD

**Issue by jimmycrequer** [Chnapy/tiled-types#7]
Hello,

I would just like to know the reason for using `any` instead of `TiledMapType` in the following definition.

```ts
export interface TiledMapAbstract<O extends TiledMapType> {
    ...
    layers: Array<TiledLayer<any>>;
    ...
}
```

Thank you!

**Comment by Chnapy** [comment]
Hi,
I guess we can use `TiledMapType` instead.
With the change I'll do for #8, generic type required by `TiledLayer` will be removed, so it'll fix this issue in the same time 👍

**Comment by Chnapy** [comment]
Fix published, please upgrade to `1.3.0` 👍
